projects/workshop/rick_morty_thiago_arend/pubspec.yaml

```
@@ -0,0 +1,94 @@
+  # package. See that file for information about deactivating specific lint
+  # rules and activating additional ones.
+  flutter_lints: ^5.0.0
+
+# For information on the generic Dart part of this file, see the
+# following page: https://dart.dev/tools/pub/pubspec
+
+# The following section is specific to Flutter packages.
+flutter:
+
+  assets:
+      - assets/
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 6

(indentation)